mos-platform/c128/link.ld

```diff
@@ -0,0 +1,25 @@
+/* C128 PRG file linker script.
+ *
+ * BASIC memory regions are freely overwritten, but any KERNAL regions are left alone.
```

Comment:
I may have missed it, but how is the basic rom disabled?

Reply:
`init-mmu.s` configures bank 0 to have the KERNAL enabled while disabling BASIC HI/LO. See http://www.oxyron.de/html/registers_mmu.html

The best option would be to disable the KERNAL ROM and do far jumps to it, which would unlock ~15.75KB of extra contiguous memory for C code.